packages/react-scripts/per-lang-loader/index.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+// This file is mostly copied from https://github.com/alienfast/i18next-loader/blob/develop/index.js
+// and adjusted to our needs for per-lang Bundling
+
+const path = require('path');
+const fs = require('fs');
+const globAll = require('glob-all');
+const loaderUtils = require('loader-utils');
+
+function enumerateLangs(dir) {
+  return fs.readdirSync(dir).filter(function(file) {
+    return fs.statSync(path.join(dir, file)).isDirectory();
+  });
+}
+
+//https://github.com/jpillora/node-glob-all#usage
+function findAll(globs, cwd) {
+  const globArray = Array.isArray(globs) ? globs : [globs];
+  return globAll.sync(globArray, { cwd, realpath: true });
+}
+
+const i18nextImport = `
+import i18n from 'i18next'
+import { addTranslations } from "@fs/zion-locale";
+
+const handleError = (locale, namespace) => {
+  console.error(\`failed to load translation - \${locale}" + "\${namespace}"\`)
```

Comment:
I added err to the signature and logged the error, which showed the stack trace, but still no the name of the component/file that failed, not sure how to add that to the logging here yet
And perhaps we should specifically ignore missing translation files to cut down on noise, or make it an option, or add it as verbose logging?